ads/google/a4a/docs/a4a-readme.md

```diff
@@ -53,3 +53,33 @@ If you are a creative agency, please express interest via [this form](https://go
 Please refer to the [Network Implementation Guide](./Network-Impl-Guide.md)
 Ad networks and ad servers can integrate with [Cloudflare](https://blog.cloudflare.com/firebolt/) who provide an AMP Ad verification services, enabling any independent ad provider to deliver faster, lighter, and more engaging ads.
 
+## Frequently Asked Questions
+
+#### Are there any AMP Ad samples?
+Yes. A number of great looking ads developed in AMP format can be found [here](https://ampbyexample.com/amp-ads/#amp-ads/experimental_ads). They use advanced components in AMP. They give the user a great experience while ensuring that the performance remains great. 
+
+#### Are there any tools to create AMP Ads?
+Yes. [Celtra](http://www.prnewswire.com/news-releases/celtra-partners-with-the-amp-project-showcases-amp-ad-creation-at-google-io-event-300459514.html) provides out of the box support for AMP ads in their ad creator platform. Other tools like Google Web Designer are also in the process of adding support. 
+
+#### How can I verify that an AMP Ad is valid?
+Depending on your development environment, there are a few options: 
```

Comment:
Worth making more explicit that AMP creatives must be valid to get preferential render?